src/ipyradiant/visualization/vis_tool.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
+
+import traitlets as T
+
+import ipywidgets as W
+from ipycytoscape import CytoscapeWidget
+from rdflib import Graph, Literal, URIRef
+from rdflib.namespace import RDF
+
+
+class CytoscapeVisualization(W.GridBox):
+    graph = T.Instance(Graph, allow_none=True)
+    cyto_widget = T.Instance(CytoscapeWidget, allow_none=True)
+    nodes = T.List()
+    click_output = T.Instance(W.Output)
+    click_output_box = T.Instance(W.VBox)
+    selected_node = T.Unicode(allow_none=True)
```

Comment:
i don't see this used anywhere... and if possible, we should plan on it being a list.